### PR TITLE
Added Sutro filter resolve #19

### LINKED ISSUE #19: Filter: Sutro

### PR DESCRIPTION
**Pull request by Stephn-R**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,9 @@
   "name": "cssgram",
   "version": "0.1.3",
   "style": "source/css/cssgram.css",
-  "dependencies": {},
+  "dependencies": {
+    "gulp-sass": "2.1.0"
```
Comment (una) [inline package.json]: this is in the dev dependencies within harp -- were you having issues?

File: site/index.html [hunk 2]
```diff
@@ -24,17 +24,25 @@ <h2 class="title--top-sub">A tiny (&lt;1kb gzipped!) library for recreating <a h
...
     <div class="demo__input-area">
       <fieldset class="demo__option-field">
         <legend>Choose a sample image:</legend>
-                  <img class="demo__option-img" src="img/atx.jpg" alt="atx image">
-                <img class="demo__option-img" src="img/bike.jpg" alt="bike image">
-                <img class="demo__option-img" src="img/cacti.jpg" alt="cacti image">
-                <img class="demo__option-img" src="img/lakegeneva.jpg" alt="lakegeneva image">
-                <img class="demo__option-img" src="img/tahoe.jpg" alt="tahoe image">
-              <br>
+
```
Comment (una) [inline site/index.html]: why add extra spaces?

**Comment by una** [comment]
There are a lot of additional files changed here -- do you mind only checking in the ones with changes? Just the sutro filter and any changes to the .twig files for test/site?

**Comment by Stephn-R** [comment]
Sure!

**Pull request closed without merging**